docs/README.md

```diff
@@ -13,9 +13,10 @@ made. We have found that following these rules results in higher quality
 Java libraries with fewer dependency conflicts and other kinds of problems. The
 list is open-ended, so new ones may be added from time to time.
 
-- [Glossary](glossary.md): Terms used in the best practices and other places in
-  cloud-opensource-java.
+## Introductory material
+- [What is a diamond dependency conflict?](what-is-a-diamond-dependency-conflict.md)
```

Comment:
This probably belongs with the glossary below.

Reply:
Why do you think the introductory material should be after the JLBPs?

Reply:
The page is for the JLBPs. Diamond dependencies are not the only thing these documents are addressing.

Reply:
Sure, but I'm going to add more content than just this article. In my mind, I think we should put the section first that people are more likely to read first. So, are they more likely to read introductory articles first or JLBPs first, assuming they land at the front page? (I wouldn't count people who land directly on a JLBP for this question, because they went there for a reason, probably following a search result or external link)

Reply:
The perhaps we should create a landing page just for that content. This page and this site is specifically for the JLBPs.

Reply:
I know we created the site for the JLBPs, but I think it is myopic to strictly limit the site to JLBPs. The content I would like to add helps people to understand the motivation for the JLBPs better and I believe it will drive more traffic to the site in a synergistic way. The site will be "heavier" and be more interlinked, meaning people will stay longer and search engines will rank it as more important. 

For the time being, I'm fine with putting the links to introductory content below the JLBPs in order to move forward. I don't want to isolate it into its own landing page yet.